# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-contain/contain-style-counters-002.html
# Commit: 86e9449aba35
# Full Hash: 86e9449aba35c7439e8a41a2804388994462c00d
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1463600
# File Overlap Count: 1
# Description:
#   Bug 1463600 - Implement CSS 'contain: style' r=emilio
#   
#   Add an implementation of CSS `contain: style`. This introduces two new
#   data structures, the ContainStyleScope and ContainStyleScopeManager.
#   
# ==============================================================================

diff -r c7d1d0cfaa6c -r 86e9449aba35 testing/web-platform/tests/css/css-contain/contain-style-counters-002.html
--- a/testing/web-platform/tests/css/css-contain/contain-style-counters-002.html	Wed Jun 22 10:30:03 2022 +0000
+++ b/testing/web-platform/tests/css/css-contain/contain-style-counters-002.html	Wed Jun 22 10:42:34 2022 +0000
@@ -5,6 +5,7 @@
   <title>CSS Containment Test: 'contain: style' and counter (with 'display: contents')</title>
 
   <link rel="help" href="https://www.w3.org/TR/css-contain-1/#containment-style">
+  <link rel=help href="https://github.com/w3c/csswg-drafts/issues/7392">
   <link rel="match" href="reference/contain-style-counters-001-ref.html">
 
   <meta content="This test checks that when an element has 'contain: style', then counters which may be affecting its subtree are reset to 0 for such scope. In this test, the div#test does not generate a principal box because of 'display: contents'. Despite that particular condition, 'contain: style' will have an effect on div#test." name="assert">